### ISSUE TITLE
Support text type

### ISSUE THREAD

**Issue by Miaourt** [relm-us/svelt-yjs#3]
Hi ! Thanks for this useful library !
Wondered if Yjs's text type could be supported too ?

**Comment by canadaduane** [comment]
It's a good question! How would you expect it to behave? I assume it would
it be for a single text field of some kind? Maybe a text area, or a
contenteditable div? How would you convey what has changed (for example,
when a user inserts a new character)? I guess we could do an automatic diff
(losing some of the semantics of the edit)... hmm.

**Comment by Miaourt** [comment]
Yeah, or simply a object that update each time it get edited ? I'm not an expert on Yjs (yet, I hope !) but it seems to be similar to the array type, with some specifics optimizations for textfield ?
